lockbox-ios/Models/KeychainManager.swift

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Security
+
+enum KeychainManagerKey: String {
+    case email, scopedKey
+}
+
+class KeychainManager {
```

Comment:
it is important to also save the FxA uid; it is used by `lockbox-datastore` as the salt for deriving the encryption key and hashing key